apps/ewallet/lib/ewallet/formatters/amount_formatter.ex

```diff
@@ -0,0 +1,20 @@
+defmodule EWallet.AmountFormatter do
+  @moduledoc """
+  A string formatter for amounts
+  """
+  def format(amount, subunit_to_unit) do
+    float_amount = amount / subunit_to_unit
+
+    float_amount
+    |> float_to_binary(to_decimals(subunit_to_unit))
+    |> String.replace_trailing(".0", "")
```

Comment:
If you swap `value` and `decimals` in `float_to_binary`, we can have something a bit cleaner:
```
subunit_to_unit
|> to_decimals()
|> float_to_binary(float_amount)
|> String.replace_trailing(".0", "")
```